examples/experimental/i3s-17-and-debug/components/tile-validator.js

```diff
@@ -76,7 +76,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  tile: null,
+  tile: {},
```

Comment:
This seems wrong. I think a value should either be a valid tile or null. Allowing a value to take on an "invalid value" seems to do more harm than good.

Isn't there a `PropTypes.optional.object`.

Reply:
returned back